src/ServiceControl.Audit.UnitTests/BodyStorage/BodyStorageEnricherTests.cs

```diff
@@ -27,10 +27,15 @@ public async Task Should_remove_body_when_above_threshold()
             var body = Encoding.UTF8.GetBytes(new string('a', maxBodySizeToStore + 1));
             var metadata = new Dictionary<string, object>();
 
-            await enricher.StoreAuditMessageBody(body, new Dictionary<string, string>(), metadata);
+            var headers = new Dictionary<string, string>
+            {
+                { Headers.MessageId, "someid" }
```

Comment:
Is this an indication that this header has become required? Or is it just for the validation of the url?

Reply:
Just for the validation of the URL. We never did before validate the URL so I added this extension to the test